plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineWtpProject.java

```diff
@@ -162,12 +162,6 @@ private void fixTestSourceDirectorySettings(IProject newProject, IProgressMonito
     // 2. Remove "src/test/java" from the Web Deployment Assembly sources.
     deployAssemblyEntryRemoveJob =
         new DeployAssemblyEntryRemoveJob(newProject, new Path("src/test/java"));
-    // Not to be affected by "NonSystemJobSuspender". Not necessary in production, but necessary
-    // for tests to work properly. (As a side note, setting this to system would have not worked
-    // if the deploy assembly update jobs triggered by the above classpath change were non-system
-    // jobs; the update jobs should be visible by "DeployAssemblyEntryRemoveJob" because it joins
-    // the update jobs before removing entries.)
-    deployAssemblyEntryRemoveJob.setSystem(true);
```

Comment:
We changed `NonSystemJobSuspender` to `ConvertJobSuspender` (https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/2112), which now suspends only `ConvertJob`s. No need to be a system job anymore.

Reply:
But that means that the job will show up in the Progress view.  I think we should keep it as a system job.

Reply:
Ah, I forgot that.